src/scout_apm/core/remote_ip.py

```diff
@@ -0,0 +1,49 @@
+
+class RemoteIp:
+    """
+    A helper class to lookup what IP the request is associated with for adding
+    Context to a request
+
+    This is not suitable to use in any security sensitive position. This is
+    spoofable by the requester.
+
+    Some of this algorithm is borrowed for Ruby on Rails' RemoteIp middleware
+    """
+
+    @classmethod
+    def lookup_from_headers(cls, headers):
+        """
+        Given a dictionary of headers (WSGI request.META for instance), look up
+        the most likely user's IP
+        """
+
+        # A single address, set by this server, returned as an Array
+        remote_addr = RemoteIp.ips_from(headers.get("REMOTE_ADDR"))
```

Comment:
Nitpick: might as well use `cls` instead of `RemoteIp`. If the class name ever changes you won't need to fix these references.